aviatrix/resource_aviatrix_splunk_logging.go

```diff
@@ -0,0 +1,146 @@
+package aviatrix
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-aviatrix/goaviatrix"
+)
+
+func resourceAviatrixSplunkLogging() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAviatrixSplunkLoggingCreate,
+		Read:   resourceAviatrixSplunkLoggingRead,
+		Update: resourceAviatrixSplunkLoggingUpdate,
+		Delete: resourceAviatrixSplunkLoggingDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"server": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Server IP",
+			},
+			"port": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Default:     -1,
+				Description: "Port number",
+			},
+			"cu_output_cfg_file_path": {
```

Comment:
We should support file contents rather than file path. 
1. File contents being changed cannot be identified since there is no diff in filename.
2. They could store the configuration in locations like S3 and the file name cannot be passed here

Customers can use the terraform file function to read the file contents

Reply:
modified to support file contents, but currently there's bug in backend(exception when using configuration file), so will test later